go-selinux/selinux.go

```diff
@@ -291,7 +314,8 @@ func selinuxEnforcePath() string {
 	return fmt.Sprintf("%s/enforce", selinuxPath)
 }
 
-func SelinuxGetEnforce() int {
+// GetEnforce returns the current SELinux mode Enforcing, Permissive, Disabled
+func GetEnforce() int {
```

Comment:
That is fine with me.  Only thing I was thinking was these match up somewhat with the libselinux.so names.  But changing these to 
EnforceMode() 
DefaultEnforceMode()
Would be more descriptive.